modules/programs/offlineimap.nix

```
@@ -90,6 +90,16 @@ in {
     programs.offlineimap = {
       enable = mkEnableOption "OfflineIMAP";
 
+      package = mkPackageOption pkgs "offlineimap" {
+        example = literalExpression ''
```

```suggestion
        example = ''
```

Looks like `mkPackageOption` adds its own `literalExpression`

---

I couldn't, for the life of me, figure out what the error in the trace was.  This was it.  I've pushed this change.